### DISCUSSION TITLE
Feature Request: Percentages

### DISCUSSION THREAD

**Discussion by gg-g** [marcusolsson/grafana-treemap-panel#36] [Ideas]
### Discussed in https://github.com/marcusolsson/grafana-treemap-panel/discussions/27

<div type='discussions-op-text'>

<sup>Originally posted by **gg-g** October  4, 2021</sup>
Thanks for the plug-in - totally use it.

I wonder if it is worth adding options to show values as percentages in the tile (see 1 in attached picture) and/or in the hover info (2).

![percentages](https://user-images.githubusercontent.com/49113408/135830167-2e51be9d-6f60-4761-99ed-0e1a24fc7f76.png)

</div>

**Comment by marcusolsson** [comment]
I think this is a good idea! I'm not sure when I'll have time to work on this, but I'd be happy to review a pull request! Similar to #33.
